docs/Makefile

```diff
@@ -17,4 +20,8 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
-	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
\ No newline at end of file
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+# Custom tasks we added (not from sphinx itself)
+serve:
+	python -m http.server ${HTTP_PORT} --directory _build/html
```

Comment:
Any particular reason for serving this vs. just opening the file directly?

Reply:
The default address to open will be the same for developers regardless of where the source is checked out

Reply:
Makes documentation a pinch easier

Reply:
Prefer if I remove vs keep?

Reply:
Sorry, I'm not following -- I usually just do `make html` then `open _build/html/index.html` and look at the local files. There doesn't seem to be any reason to use a backend server here.

Reply:
On WSL2:

```
> open
zsh: command not found: open
```

The user may not necessarily have a CLI command to launch a file.

Reply:
I think it is fine to include though, just wondering if I'm missing something by directly opening the file. 😄

Reply:
What I mean to say is:

If you are writing developer docs, e.g. https://vcspull.git-pull.com/developing.html#documentation

It's not easy to convey to the developer how to launch a browser to a file location. It depends on their environment, habits, and what they already know.  

It's common in many places to launch an HTTP server while developing, even if it's static files (webpack dev server)

I personally don't know any commands to.  But if you give them a URL, and say open http://0.0.0.0:8038, those instructions work for everyone.